### PR TITLE
Ball Tree Algorithm and it's test

### PR DESCRIPTION
**Pull request by anishkabhatia**

### CODE REVIEW

File: datascientist/model/classification/skl/linear_model/balltree.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+from sklearn.neighbors import BallTree
+from sklearn.metrics import f1_score
+from sklearn.metrics import jaccard_score
+from sklearn.metrics import accuracy_score
+import numpy as np
+
+def _BallTree(X, leaf_size=40, metric='minkowski', **kwargs):
```
Comment (sidhu1012) [inline datascientist/model/classification/skl/linear_model/balltree.py:8]: This is not the right way, please go through other models once.Will merge if looks good irrespective of Travis

Comment (anishkabhatia) [reply datascientist/model/classification/skl/linear_model/balltree.py:8]: > This is not the right way, please go through other models once. Will merge if looks good irrespective of Travis

other models as in? I can't understand

Comment (anishkabhatia) [reply datascientist/model/classification/skl/linear_model/balltree.py:8]: @sidhu1012  could you elaborate? What is the right way?

Comment (sidhu1012) [reply datascientist/model/classification/skl/linear_model/balltree.py:8]: Please look at other classification models.

Comment (anishkabhatia) [reply datascientist/model/classification/skl/linear_model/balltree.py:9]: @sidhu1012 I formatted the model. Please have a look at it and let me know if any changes are required!

**Pull request closed without merging**